lib/cli/commands/config.js

```diff
@@ -0,0 +1,88 @@
+import chalk from "chalk";
+import process from "node:process";
+import baseMiddleware from "../middlewares/base.js";
+
+const configCommand = {
+	command: "config",
+	describe: "Get and set UI5 Tooling configuration options",
+	middlewares: [baseMiddleware],
+	handler: handleConfig
+};
+
+configCommand.builder = function(cli) {
+	return cli
+		.demandCommand(1, "Command required. Available commands are 'set', 'get' and 'list'")
+		.command("set <key> [value]",
+			"Set the value for a given configuration key. Provide no unset the configuration", {
+				handler: handleConfig,
+				builder: noop,
+				middlewares: [baseMiddleware],
+			})
+		.command("get <key>", "Get the value for a given configuration key", {
+			handler: handleConfig,
+			builder: noop,
+			middlewares: [baseMiddleware],
+		})
+		.command("list", "Display the current configuration", {
+			handler: handleConfig,
+			builder: noop,
+			middlewares: [baseMiddleware],
+		})
+		.example("$0 set mavenSnapshotEndpointUrl http://example.com/snapshots/",
+			"Set a value for the mavenSnapshotEndpointUrl configuration")
+		.example("$0 set mavenSnapshotEndpointUrl",
+			"Unset the current value of the mavenSnapshotEndpointUrl configuration");
+};
+
+function noop() {}
+
+async function handleConfig(argv) {
+	const {_: commandArgs, key, value} = argv;
+	const command = commandArgs[commandArgs.length - 1];
+
+	const {default: Configuration} = await import( "@ui5/project/config/Configuration");
+	const allowedKeys = ["mavenSnapshotEndpointUrl"];
```

Comment:
Shouldn't this information come from the Configuration class instead of hard-code it here?
It would allow us to add new options without having to adjust the CLI project.

Reply:
True. That's a good idea we should follow-up on 👍